examples/flutter_view/macos/Runner/MainFlutterWindow.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Cocoa
+import FlutterMacOS
+
+class MainFlutterWindow: NSWindow {
+  override func awakeFromNib() {
+    let windowFrame = self.frame
+    let storyboard = NSStoryboard(name: NSStoryboard.Name("Main"), bundle: Bundle.main)
+    /*
+    `flutter create --target=macos` uses this class (`self`) as an entrypoint
+    for drawing on a surface. The line below intercepts that and uses
+    the storyboard from `Main.storyboard`.
+    */
```

Comment:
Similar comment for Swift comments.

https://google.github.io/swift/#non-documentation-comments